src/fragmentarium/ui/image-annotation/annotation-tool/mapTokensToAnnotationTokens.test.ts

```diff
@@ -164,9 +192,9 @@ it('', () => {
       suffixModifier: null,
       type: 'LineNumber',
     },
-  })
+  }
   const text = new Text({
-    lines: [at.surface, textLine, dollar.singleRuling],
+    lines: [at.surface, (textLine as unknown) as TextLine, dollar.singleRuling],
```

Comment:
Do not do this. `textLine` is not a `TextLine`. Not sure what you are after? It was a `TextLine`. You changed it, but cast (incorrectly) it back to a `TextLine`

Reply:
i changed the token interface